packages/iov-ethereum/src/ethereumconnection.ts

```diff
@@ -133,8 +150,11 @@ export class EthereumConnection implements AtomicSwapConnection {
       this.socket.connect();
     }
 
+    const atomicSwapEtherContractAddress =
+      (options && options.atomicSwapEtherContractAddress) || constants.atomicSwapEtherContractAddress;
```

Comment:
Can we remove `atomicSwapEtherContractAddress` from constants? It is never a @iov/ethereum-level constant but always network specific. If unset, atomicSwapEtherContractAddress can be `undefined`.

Reply:
Definitely, that's been the plan all along, I just needed somewhere to put it while I got all the other bits sorted out.